SysKit.XCellKit.SampleApp/Demos/LargeTableStreamingBase.cs

```diff
@@ -1,14 +1,15 @@
-﻿using System.Collections.Generic;
-using System.Drawing;
+﻿using SkiaSharp;
+using System.Collections.Generic;
 
 namespace SysKit.XCellKit.SampleApp.Demos
 {
     abstract class LargeTableStreamingBase : DemoBase
     {
         protected const int RowsToStream = 800000;
-        static Font _font = new Font(new FontFamily("Calibri"), 11);
+        private static SKFont _font = new SKFont(SKTypeface.FromFamilyName("Calibri"), 11);
```

Comment:
Calibri could be a linux only font

Reply:
Calibri is a windows only font. It's referenced only in tests and demo app, so I would leave it as it is.